backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/UserAccountCreationController.java

```diff
@@ -134,8 +135,13 @@ public void enrollEmailMfa(@RequestBody EnrollMfaRequest requestBody, HttpServle
    * @throws OktaAuthenticationFailureException if Okta cannot enroll the user in MFA.
    */
   @GetMapping("/authenticator-qr")
-  public void getAuthQrCode(@RequestBody EnrollMfaRequest requestBody, HttpServletRequest request) {
-    // WIP: doesn't interact with Okta yet.
+  public String getAuthQrCode(
```

Comment:
Should we return a json body with a property for the qr code link?  If it's just a text response with the string then it'll be more difficult to change if we ever need to (for example, adding something to the response would become a breaking change).

Reply:
Sounds good, changed to a JSONObject. (In `UserAccountCreationController` I'm still just sending the QR code link to the frontend though.)